sensor_msgs/Imu imu
sensor_msgs/MagneticField mag
Euler euler
geometry_msgs/Vector3Stamped free_acceleration
sensor_msgs/FluidPressure pressure
